Runtime/AvatarTagComponent.cs

```diff
@@ -55,4 +58,15 @@ public virtual void ResolveReferences()
         {
         }
     }
+    
+#if !MA_VRCSDK3_AVATARS
+
+    /**
+     * Placeholder of VRC.SDKBase.IEditorOnly for environments without VRCSDK
+     */
+    interface IEditorOnly
+    {
+    }
+
+#endif
```

Comment:
MEMO: should this go to ndmf? https://github.com/anatawa12/AvatarOptimizer/pull/627/files